### PR TITLE
ceph-volume: support GPT and other deployed OSDs

### PR DESCRIPTION
**Pull request by alfredodeza**
* creates the `simple` sub command
* parses both paths and devices
* creates JSON config files in /etc/ceph/osd/
* triggers on system boot
* disables ceph-disk systemd interaction
* can activate and take over after a scan
* supports GPT and LVM devices

Adds unit tests and docs, functional testing to follow once this is merged.

### CODE REVIEW

File: doc/ceph-volume/simple/activate.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+To activate parsed OSDs::
+
+    ceph-volume simple activate 0 6cc43680-4f6e-4feb-92ff-9c7ba204120e
+
+The above command will assume that a JSON configuration will be found in::
+
+    /etc/ceph/osd/0-6cc43680-4f6e-4feb-92ff-9c7ba204120e.json
+
+
+requiring uuids
```
Comment (andrewschoen) [inline doc/ceph-volume/simple/activate.rst]: Are you allowing a path to the json file too? If so, that should be documented as well.

Comment (alfredodeza) [reply doc/ceph-volume/simple/activate.rst]: This has been added, and also added an example to the help output in ``activate``

**Comment by andrewschoen** [comment]
jenkins test ceph-volume tox

**Comment by andrewschoen** [comment]
jenkins test ceph-volume tox

**Comment by andrewschoen** [comment]
jenkins test ceph-volume tox

**Comment by alfredodeza** [comment]
jenkins test ceph-volume tox

**Pull request merged**